doc/sphinx/addendum/omega.rst

```diff
@@ -5,6 +5,19 @@ Omega: a solver for quantifier-free problems in Presburger Arithmetic
 
 :Author: Pierre Crégut
 
+.. warning::
+
+   The :tacn:`omega` tactic is about to be deprecated in favor of the
+   :tacn:`lia` tactic, which is a more complete tactic to solve the
+   same kind of arithmetic goals.  Work is in progress to make sure
```

Comment:
Maybe this could be framed more in terms "the deprecation of omega in favor of lia serves the goal of consolidating the arithmetic solving capabilities of Coq into a single engine; moreover, lia is in general more powerful than omega"

"Unfortunately, swithching from omega to lia cannot often be done in a seamless way due to lia solving more goals than omega; scripts that don't take this into account will indeed break" etc...

Reply:
So far the main cause of breakage from `omega` to `lia` seems to be that lia is more powerful, above I tried to present that info to users ,however IMHO we can do better and even suggest some robust tactic setup to alleviate that.

Reply:
Thanks for the suggestions!

Reply:
I would make it more precise: the scripts which break are the ones which were relying on incompleteness bugs of `omega`, i.e. linear arithmetic goals that would not be solved by `omega`.